jsinterop-ts-defs-processor/src/main/java/com/vertispan/tsdefs/model/TsClass.java

```diff
@@ -33,7 +33,7 @@ public class TsClass implements IsType {
   private TsClass superClass;
   private final List<TsInterface> interfaces = new ArrayList<>();
   private final List<TsModifier> modifiers = new ArrayList<>();
-  private final Set<TsProperty> properties = new LinkedHashSet<>();
+  private final Set<TsVariable> properties = new LinkedHashSet<>();
```

Comment:
since this is TsClass, shouldnt this still be typed as TsProperty (as it is named `properties`)?